### ISSUE TITLE
atomic install --system issue

### ISSUE THREAD

**Issue by aveshagarwal** [projectatomic/atomic#593]
When installing imported images by `atomic install --system` outputs following error:

`mmap: Invalid argument`

@giuseppe

**Comment by giuseppe** [comment]
can you paste the output of atomic --debug install --system?

I've never seen something like that with atomic

**Comment by aveshagarwal** [comment]
```
# ./atomic --debug uninstall etcd3
systemctl stop etcd3
systemctl disable etcd3
systemd-tmpfiles --remove /etc/tmpfiles.d/etcd3.conf
mmap: Invalid argument
```

**Comment by aveshagarwal** [comment]
I see that with atomic install also i.e. without --system too

**Comment by rhatdan** [comment]
Is this something from inside of the etcd container?

**Comment by yuqi-zhang** [comment]
I have also not been able to reproduce this. Does this still occur @aveshagarwal ? Does this happen with all containers?

**Comment by rhatdan** [comment]
Fixed in atomic-1.13
